src/components/List.js

```diff
@@ -51,6 +54,48 @@ const List = ({ firestore }) => {
     setInputText('');
   };
 
+  //* Created two lists
+  const inactiveList = [];
+  const activeList = [];
+
+  //* Used this function to divide active and inactive items
+  filteredList.map(item => {
+    let formatedLastPurchaseDate = dayjs.unix(
+      item.lastPurchasedDate['seconds'],
+    );
+    let formatedToday = dayjs();
+    let difference = formatedToday.diff(formatedLastPurchaseDate, 'd');
+    if (
+      item.numberOfPurchases <= 1 ||
+      parseInt(difference) >= item.nextPurchase * 2
```

Comment:
I think there are some weird use cases where parseInt converts to NaN. It may be safer to specify the base as the second argument - `parseInt(difference, 10)`

Reply:
Ah, good call, @lilyhoratio. `parseInt` will return `NaN` when the `redix` (that second argument you mentioned) is smaller than 2 or bigger than 36, or the first non-whitespace character cannot be converted to a number. You can [get some more deets here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) too, but I'd def recommend passing a `redix`

Reply:
Will `difference` ever contain something other than a number (albeit in string form)? If we know that it will always be a value that can be coerced into a number, another way to do that is to use the `+` operator, as follows:

    +difference >= item.nextPurchase * 2

For the record, `item.nextPurchase` is (I think) being coerced to a number by the multiplication operator as well.